### DISCUSSION TITLE
Trigger workflow when Two approver approved the Pull request

### DISCUSSION THREAD

**Discussion by anupkumardh** [community/community#29007] [Actions] [answered]
One of my requirement to trigger the workflow when 2 approver approve the pull request, right now we have pull_request_review  trigger and code like below but this will trigger the workflow when any one of the code owner approve the pull request

`jobs:
  test_job:
    if: |
      github.event.review.state == 'approved' && 
      github.event.pull_request.base.ref == 'feature'`

Is there any way to trigger the workflow only when two people approved, As per my research i couldn't find anything any pointers are much appreciated

**Comment by thuongtruong1009** [comment, marked as answer]
Hi @anupkumardh 
* You can try ```if:  startsWith(Github.head_ref, 'releases/')``` instead of ```github.event.pull_request.base.ref == 'feature'```
* You can try to split this action into 2 conditional parts run in order
* Reference more at [Event trigger workflow](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target)
